LoopKitUI/View Controllers/ChartsTableViewContainer.swift

```diff
@@ -0,0 +1,189 @@
+//
+//  ChartsTableViewController2.swift
+//  LoopKitUI
+//
+//  Copyright © 2017 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+import HealthKit
+import os.log
+
+public protocol ChartsTableViewControllerDelegate: class {
```

Comment:
This protocol adds the needed logic for the charts to update when the glucose unit changes value